src/VisualStudio/Core/Def/Implementation/AbstractVsTextViewFilter`2.cs

```diff
@@ -32,7 +33,20 @@ internal abstract class AbstractVsTextViewFilter<TPackage, TLanguageService> : A
             _languageService = languageService;
         }
 
-        public virtual int GetDataTipText(TextSpan[] pSpan, out string pbstrText)
+        int IVsTextViewFilter.GetDataTipText(TextSpan[] pSpan, out string pbstrText)
```

Comment:
or do we omit access modifiers for internal?

---
In reply to: [134579697](https://github.com/dotnet/roslyn/pull/21651#discussion_r134579697) [](ancestors = 134579697)